packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -110,7 +130,8 @@ class _SnackBarActionState extends State<SnackBarAction> {
       _haveTriggeredAction = true;
     });
     widget.onPressed();
-    Scaffold.of(context).hideCurrentSnackBar(reason: SnackBarClosedReason.action);
+    Scaffold.of(context)
+        .hideCurrentSnackBar(reason: SnackBarClosedReason.action);
```

Comment:
nit: no need to wrap

i'm guessing that dartfmt was accidentally run against this file